.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
idea files should not be in git. you need to add it to .gitignore file. I'll show you tomorrow